packages/@o3r/schematics/schematics/eslint/index.ts

```diff
@@ -0,0 +1,13 @@
+import type { Rule } from '@angular-devkit/schematics';
+import { eslintFix } from '@o3r/schematics';
+import type { EslintSchematicsSchema } from './schema';
+
+/**
+ * Eslint Schematics
+ */
+export function eslint(options: EslintSchematicsSchema): Rule {
```

Comment:
This is really similar to [@angular-eslint/builder](https://github.com/angular-eslint/angular-eslint).
Is there is reason to have our custom schematic to run eslint?